src/gaurav/Assignment24/MiddleCharacter.java

```diff
@@ -0,0 +1,19 @@
+package gaurav.Assignment24;
+
+public class MiddleCharacter {
+	public static void main(String[] args) {
+		String[] arr= {"Techno","Hello","Credits"};
+		 for(int i=0;i<arr.length;i++) {
```

Comment:
take reference of below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1109/files